frontend_new/src/App.tsx

```diff
@@ -19,32 +19,48 @@ function App() {
     document.body.style.backgroundColor =
       themeChoice.palette.background.default;
   }, [themeChoice]);
-  const themeSwitch = () => {
+  const themeChange = () => {
     if (themeChoice.palette.type === "dark") {
       setThemeChoice(light);
     } else {
       setThemeChoice(dark);
     }
```

Comment:
**[Minor]** Might as well use a ternary operator. Not really a problem though.

Reply:
resolved in latest commit